data/planet/-hosted_on_ocamlorg/opam/signing-the-opam-repository.md

```diff
@@ -6,6 +6,7 @@ description: "Proposal for a secure distribution of OCaml packages"
 source:
   name: Opam blog
   url: ""
+  tag: "personnal blog"
```

Comment:
Pretty sure that the opam blog would not qualify as someone's personal blog. It's probably more of an "ocaml platform blog" blog, or an "open source project" kind of blog